docs/learning/backend/Workspace.md

```diff
@@ -1,475 +1,222 @@
 
 ```ts
-    "cloud/containers": {
-      "type": "array",
-      "description": "array of cloud containers",
-      "cumulative": true,
-      "items": {
-        "type": "object",
-        "required": [
-          "name",
-          "containerId"
-        ],
-        "properties": {
-          "name": {
-            "type": "string",
-            "description": "the alias name of this cloud container"
-          },
-          "baseUri": {
-            "type": "string",
-            "description": "the baseUri for the container, without trailing slash (e.g., https://myAcct.blob.core.windows.net)"
-          },
-          "containerId": {
-            "type": "string",
-            "description": "the containerId of this cloud container"
-          },
-          "storageType": {
-            "type": "string",
-            "description": "one of: 'azure', 'aws', 'google'"
-          },
-          "isPublic": {
-            "type": "boolean",
-            "description": "whether the cloud container is public (doesn't require authentication)",
-            "default": false
-          }
-        }
-      }
-    },
-    "workspace/databases": {
-      "type": "array",
-      "cumulative": true,
-      "description": "array of workspace databases",
-      "items": {
-        "type": "object",
-        "required": [
-          "name",
-          "dbName",
-          "containerName"
-        ],
-        "name": {
-          "type": "string",
-          "description": "the alias name of the workspace database"
-        },
-        "properties": {
-          "dbName": {
-            "type": "string",
-            "description": "the name of the database within its cloud container"
-          },
-          "containerName": {
-            "type": "string",
-            "description": "the cloud container name"
-          },
-          "version": {
-            "type": "string",
-            "description": "the (semver) range of acceptable versions"
-          },
-          "includePrerelease": {
-            "type": "boolean",
-            "description": "include prerelease version as acceptable versions"
-          },
-          "priority": {
-            "type": "number",
-            "description": "for sorted databases, higher values are searched first"
-          },
-          "prefetch": {
-            "type": "boolean",
-            "description": "if true, pre-fetch all of the data from the cloud in the background"
-          }
-        }
-      }
-    },
+[[include:WorkspaceExamples.CreateWorkspaceDb]]
 ```
 
-For example:
+Since every [WorkspaceDb]($backend) must reside inside a [WorkspaceContainer]($backend), we must first create a container. Creating a container also creates a default `WorkspaceDb`. In the example above, we set up the default `WorkspaceDb` to be our as-yet empty "cornus" tree database.
```

Comment:
nit: `Since every [WorkspaceDb]($backend) must reside inside a [WorkspaceContainer]($backend), we must first create a container.` should probably appear before the example.

Reply:
Moved it above.